keyboards/satan/keymaps/jarred/config.h

```diff
@@ -0,0 +1,8 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+ 
+#include "../../config.h"
+ 
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
This line is not actually needed, and should be removed.

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.